src/20_Components/amp-experiment.html

```diff
@@ -0,0 +1,80 @@
+<!---{
+    "experiment": true,
+    "component": "amp-experiment"
+  }--->
+<!-- 
+  #### Introduction
+
+  [amp-experiment](https://github.com/ampproject/amphtml/blob/master/extensions/amp-experiment/amp-experiment.md) allows to perform user experience experiments on an AMP document and collect resulting data.
```

Comment:
@rudygalfi I think you mentioned somewhere that we want to use the phrase "user experience experiment" to prevent the wrong implication of "only 2 variants are allowed"?